documents/Project_idea_document.md

```diff
@@ -45,6 +45,47 @@ The possibility to sell the ideas to companies was mentioned. I prefer the idea
 Also, the idea of ads was shot down by the professor, but I believe that if you can show ads that are relevant and non-intrusive it is not a bad idea. For example, there is an popular idea to make a virtual pet dragon app. You can have a web-crawler search for a similar app that is already available (similar but most likely not exactly the same) and offer the makers of the app some adspace on the site. The ad would be relevant to what the user is looking at. The idea on the site can still be elaborated on and might turn into something quite different.
 
 ####Requirements and Specifications
+
+#####a)	User Characteristics
+The main users that the application is created for are both experienced and non-experienced users; it can be used by average users that have ideas that they would like to be made but do not have the required time/skills/money for it. The general users that the application addresses are listed below:
```

Comment:
You list the user groups as "experienced" and "non-experienced" in the first sentence but then list three types below. Maybe it would be good to make these consistent?